x-pack/agent/_meta/agent.fleet.yml

```diff
@@ -2,22 +2,7 @@
 # Beats is configured under Fleet, you can define most settings
 # from the Kibana UI. You can update this file to configure the settings that
 # are not supported by Fleet.
-outputs:
```

Comment:
removing this?

Reply:
The output configuration is coming from Fleet so no need to have it in this configuration file :)